arwen/hosttest/tools/graphviz_tool.go

```diff
@@ -1,178 +1,95 @@
+	// callGraph := test.CreateGraphTestOneAsyncCallbackFailCrossShard()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsSecondCallbackFailLocalCross()
+	// callGraph := test.CreateGraphTestAsyncCallbackIndirectFailCrossShard()
+	// callGraph := test.CreateGraphTestSyncCalls()
+	// callGraph := test.CreateGraphTestSyncCalls2()
+	// callGraph := test.CreateGraphTestOneAsyncCall()
+	// callGraph := test.CreateGraphTestOneAsyncCallCrossShard()
+	// callGraph := test.CreateGraphTestOneAsyncCallCrossShard2() //!
+	// callGraph := test.CreateGraphTestOneAsyncCallNoCallbackCrossShard()
+	// callGraph := test.CreateGraphTestOneAsyncCallFailNoCallbackCrossShard()
+	// callGraph := test.CreateGraphTestTwoAsyncCalls()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsOneFail()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsLocalCross()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsCrossLocal()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncSecondFail()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncLocalCross()
+	// callGraph := test.CreateGraphTestCallbackCallsSync()
+	// callGraph := test.CreateGraphTestSyncAndAsync1()
+	// callGraph := test.CreateGraphTestSyncAndAsync2()
+	// callGraph := test.CreateGraphTestSyncAndAsync3()
+	// callGraph := test.CreateGraphTestSyncAndAsync6()
+	// callGraph := test.CreateGraphTestSyncAndAsync7()
+	// callGraph := test.CreateGraphTestSyncAndAsync8()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsCrossShard()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsFirstCallbackFailCrossShard()
+	// callGraph := test.CreateGraphTestSyncCallsFailPropagation()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsFirstFail()
+	// callGraph := test.CreateGraphTestTwoAsyncCallsFirstFailLocalCross()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncFirstNoCallbackLocalCross()
+	// callGraph := test.CreateGraphTestOneAsyncCallCustomGasLocked()
+
+	/*
+		multi lvl of async calls
+	*/
+	// callGraph := test.CreateGraphTestAsyncCallsAsync()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncLocalCross()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncCrossShard()
+	// callGraph := test.CreateGraphTestAsyncsOnMultiLevelFail1()
+	// callGraph := test.CreateGraphTestCallbackCallsAsyncCrossCross()
+	// callGraph := test.CreateGraphTestAsyncCallsCrossShard6()
+	// callGraph := test.CreateGraphTestAsyncCallsCrossShard7()
+	callGraph := test.CreateGraphTestSyncAndAsync5()
+	// callGraph := test.CreateGraphTestDifferentTypeOfCallsToSameFunction()
+	// callGraph := test.CreateGraphTestCallbackCallsAsyncLocalLocal()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncSecondFail()
+	// callGraph := test.CreateGraphTestCallbackCallsAsyncCrossLocal()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncSecondCallbackFailCrossShard()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncBothCallbacksFailLocalCross()
+	// callGraph := test.CreateGraphTestAsyncCallsAsyncSecondCallbackFailLocalCross()
```

Comment:
Consider converting these lines into a separate function `getCallGraph(graphName)` which has a large `switch` inside it.